src/include/nodes/plannodes.h

```diff
@@ -76,7 +70,12 @@ typedef struct PlannedStmt
 
 	CmdType		commandType;	/* select|insert|update|delete|utility */
 
-	PlanGenerator	planGen;		/* optimizer generation */
+	/*
+	 * If the plan is done by an extension, the field below should contain
+	 * the name of the extension planner. Should be NULL if planning is done
+	 * by Postgres planner.
+	 */
+	const char	*plannerName;
```

Comment:
What about moving the plannerName field to the end of the PlannedStmt structure, because most of GPBD-specific field are located here? It will be useful to reduce diff with PostgreSQL